<launch>

  <node name="amr_twist_keyboard" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" output="screen">
    <remap from="cmd_vel" to="/amr/mobile_base_controller/cmd_vel" />
  </node>

</launch>